### ISSUE TITLE
UpgradeMODX 2.3.1 and MODX 2.8.3

### ISSUE THREAD

**Issue by Jako** [BobRay/UpgradeMODX#74]
The current version of UpgradeMODX shows the following widget in the frontend:

<img width="544" alt="Screenshot" src="https://user-images.githubusercontent.com/148371/163353259-0d54ab40-1273-447d-9231-05e528ed2f9a.png">

Version 3 is disallowed, this is fine. But the widget shows the begin upgrade button, even if there is no further version in the 2.x branch.

**Comment by Camo30** [comment]
In my opinion "Upgrade Available" and "Latest Version: 3.0.0-pl" should not be shown when `ugm_show_modx3` is disabled.

**Comment by BobRay** [comment]
I agree, but it may be difficult given the way that the current version is identified. I'll try to find time to look into it.

**Comment by BobRay** [comment]
I believe this is fixed. Reopen if it's still a problem.
